src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -444,12 +502,17 @@ void doLastBlock(byte[] in, int inOfs, int len, byte[] out, int outOfs,
      * @return the number of bytes placed into the <code>out</code> buffer
      */
     int encrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
-        ArrayUtil.blockSizeCheck(len, blockSize);
-
```

Comment:
With the removal of this blockSizeCheck, the javadoc exception statement above at line 500 becomes obsolete?